modin/test/test_partition_api.py

```diff
@@ -33,10 +33,10 @@
     get_func = ray.get
     FutureType = ray.ObjectRef
 elif Engine.get() == "Dask":
-    from distributed.client import default_client
+    from modin.core.execution.dask.common.task_wrapper import DaskWrapper
     from distributed import Future
 
-    put_func = lambda x: default_client().scatter(x)  # noqa: E731
+    put_func = lambda x: DaskWrapper.put(x)  # noqa: E731
     get_func = lambda x: x.result()  # noqa: E731
```

Comment:
shouldn't this be changed as well?
```suggestion
    put_func = lambda x: DaskWrapper.put(x)  # noqa: E731
    get_func = lambda x: DaskWrapper.materialize(x)  # noqa: E731
```